agenta-web/src/lib/helpers/openapi_parser.ts

```diff
@@ -4,7 +4,9 @@ import {GenericObject, Parameter} from "../Types"
 
 const getBodySchemaName = (schema: GenericObject): string => {
     return (
-        schema?.paths?.["/generate"]?.post?.requestBody?.content["application/json"]?.schema["$ref"]
+        schema?.paths?.["/generate"]?.post?.requestBody?.content?.["application/json"]?.schema?.[
+            "allOf"
```

Comment:
This does not work for me. I created a PR that fixes it (using chatGPT). I am not sure why and how, just that it fixes it, please look into it.